insights/config/specs.py

```diff
@@ -65,6 +65,9 @@
     "df_-alP"                   : CommandSpec("/bin/df -alP"),
     "df_-li"                    : CommandSpec("/bin/df -li"),
     "dig"                       : CommandSpec("/usr/bin/dig +dnssec . DNSKEY"),
+    "dig_noedns"                : CommandSpec("/usr/bin/dig +noedns . SOA"),
+    "dig_edns"                  : CommandSpec("/usr/bin/dig +edns=0 . SOA"),
+    "dig_dnssec"                : CommandSpec("/usr/bin/dig +dnssec . SOA"),
```

Comment:
@skontar would you mind to swap the order of `dig_noedns` and `dig_dnssec` to keep the specs in sorted order.